test/common_quantized.py

```diff
@@ -65,8 +65,12 @@ def _calculate_dynamic_qparams(X, dtype):
 
 @contextmanager
 def enable_mobile_quantized_engine():
-    torch.backends.quantized.engine = torch.qnnpack
+    torch.backends.quantized.engine = "qnnpack"
     try:
         yield
     finally:
-        torch.backends.quantized.engine = torch.fbgemm
+        qengines = torch._C._supported_qengines()
```

Comment:
should we have another wrapper around `_supported_qengines` which returns a list of strings to user?

Reply:
Yeah, it will make everything string which is more consistent.